modules/op-resolver-git-config-multiple-providers.adoc

```diff
@@ -0,0 +1,73 @@
+// This module is included in the following assemblies:
+// * create/remote-pipelines-tasks-resolvers.adoc
+// *openshift_pipelines/remote-pipelines-tasks-resolvers.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="op-resolver-git-config-multiple-providers_{context}"]
+= Configuring multiple Git providers
+
+You can configure multiple Git providers, or you can add multiple configurations for the same Git provider, to use in different task runs and pipeline runs.
+
+Add details in the `TektonConfig` custom resource (CR) with your unique identifier key prefix.
+
+.Procedure
+
+. Edit the `TektonConfig` CR, by running the following command:
++
+[source,terminal]
+----
+$ oc edit TektonConfig config
+----
+
+. In the `TektonConfig` CR, edit the `pipeline.git-resolver-config` spec:
++
+[source,yaml]
+----
+apiVersion: operator.tekton.dev/v1alpha1
+kind: TektonConfig
+metadata:
+  name: config
+spec:
+# ...
+  pipeline:
+    git-resolver-config:
+      # configuration 1 # <1> 
+      fetch-timeout: "1m"
+      default-url: "https://github.com/tektoncd/catalog.git"
+      default-revision: "main"
+      scm-type: "github"
+      server-url: ""
+      api-token-secret-name: ""
+      api-token-secret-key: ""
+      api-token-secret-namespace: "default"
+      default-org: ""
+      # configuration 2 # <2> 
+      test1.fetch-timeout: "5m"
+      test1.default-url: ""
+      test1.default-revision: "stable"
+      test1.scm-type: "github"
+      test1.server-url: "api.internal-github.com"
+      test1.api-token-secret-name: "test1-secret"
+      test1.api-token-secret-key: "token"
+      test1.api-token-secret-namespace: "test1"
+      test1.default-org: "tektoncd"
+      # configuration 3 # <3>
+      test2.fetch-timeout: "10m"
+      test2.default-url: ""
+      test2.default-revision: "stable"
+      test2.scm-type: "gitlab"
+      test2.server-url: "api.internal-gitlab.com"
+      test2.api-token-secret-name: "test2-secret"
+      test2.api-token-secret-key: "pat"
+      test2.api-token-secret-namespace: "test2"
+      test2.default-org: "tektoncd-infra"
+# ...
+----
+<1> The default configuration to use if no `configKey` key is provided or the key is provided with the `default` value.
+<2> The configuration used if the `configKey` key is passed with the `test1` value.
+<3> The configuration used if the `configKey` key is passed with the `test2` value.
++
+[WARNING]
+====
+The `configKey` values with the `.` symbol are not supported. If you try to pass the `configKey` value with the `.` symbol, the `TaskRun` or `PipelineRun` resource where you passed the value fails to run.
```

Comment:
```suggestion
`configKey` values with the `.` symbol are not supported. If you try to pass a `configKey` value that contains the `.` symbol, the `TaskRun` or `PipelineRun` resource where you passed the value fails to run.
```

"The" implies specific, enumerated values IMO. If that's the case, this suggestion is invalid.